6.6/scripts/oom_solr.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This is just a wrapper to make it actually work in the start script - because
+# the whitespaces of belows call cause havoc otherwise:
+#. /opt/solr/bin/oom_solr.sh ${SOLR_PORT:-8983} /opt/solr/server/logs
+
+SOLR_LOGS_DIR=/opt/solr/server/logs
```

Comment:
No need to set that; it's already set in the Dockerfiles

Reply:
That's only the case for versions >= 8.
The only other variable available that resembles that for versions <8 is the optional `MY_SOLR_LOGS_DIR`.
I've changed the >= 8 script to not re-define `SOLR_LOGS_DIR` and the <8 script to use `MY_SOLR_LOGS_DIR` if available and fallback to `/opt/solr/server/logs` otherwise.